Constitution.txt

```diff
@@ -1,490 +1,264 @@
+9: In Suits at common law, where the value in controversy shall exceed the value of five days' labor of any involved Party, the right of trial by jury shall be preserved, and no fact tried by a jury, shall be otherwise re-examined in any Court of these United States, than according to the rules of the common law.
 
-Amendment XXV  affects 9 (Amendment 25 - Presidential Vacancy, Disability, and Inability)
-1: In case of the removal of the President from office or of his death or resignation, the Vice President shall become President.
+10: Excessive bail shall not be required, nor excessive fines imposed, nor cruel or unusual punishments inflicted.
 
-2: Whenever there is a vacancy in the office of the Vice President, the President shall nominate a Vice President who shall take office upon confirmation by a majority vote of both Houses of Congress.
+11: Neither slavery nor involuntary servitude, except as a punishment for crime whereof the party shall have been duly convicted, shall exist within these United States, or any place subject to their jurisdiction; and no Person shall be considered the Property of any other Person.
 
-3: Whenever the President transmits to the President pro tempore of the Senate and the Speaker of the House of Representatives his written declaration that he is unable to discharge the powers and duties of his office, and until he transmits to them a written declaration to the contrary, such powers and duties shall be discharged by the Vice President as Acting President.
+Article VII (Article 7 - Definition of Citizenship)
+1: All persons naturalized in these United States, born to natural Citizens of these United States, or resident in and subject to any State joining the Union at time of joining, are Citizens of these United States and of the State wherein they reside. No State shall make or enforce any law which shall abridge the privileges or immunities of citizens of these United States; nor shall any State deprive any person of life, liberty, or property, without due process of law; nor deny to any Person within its jurisdiction the equal protection of the laws.
 
-4: Whenever the Vice President and a majority of either the principal officers of the executive departments or of such other body as Congress may by law provide, transmit to the President pro tempore of the Senate and the Speaker of the House of Representatives their written declaration that the President is unable to discharge the powers and duties of his office, the Vice President shall immediately assume the powers and duties of the office as Acting President.
+2: The eligibility of a Citizen to vote in the election of a Representative, or of an Elector, or of a member of the Legislature of the State in which they reside, shall be determined according to the law of the State; and such law may not take into consideration factors which the Citizen has no ability to change, or has not caused to himself, except for the case of Age.
```

Comment:
Here I update the 15th amendment, to leave the power to decide Voter requirements with the States, rather than delegating it to Congress.

I also remove the need to list identity categories, by simply defining the category of "identity category." I specifically allow for restrictions based on Age, as an exception, because States should decide the age of maturity for electoral purposes.